tests/src/openshift-local-extension.spec.ts

```diff
@@ -112,7 +114,7 @@ test.describe.serial('Red Hat OpenShift Local extension verification', () => {
   });
 
   test.describe.serial('Red Hat OpenShift Local extension handling', () => {
-    test('Extension can be disabled', async ({ navigationBar }) => {
+    test.fail('Extension can be disabled -- Preferences value should be removed after extension removal, but isn\'t, BUG #393', async ({ navigationBar }) => {
```

Comment:
I would divide the test into two parts. The first part would basically stay the same, then I would add test.fail part with failure exception to just checking the preferences section removal. This way it will not happen that we have false negative test result -> ie. resources card is still present after disabling the extension and we think the failure was because of later checks.